### ISSUE TITLE
rake spec fails on new app

### ISSUE THREAD

**Issue by danil-z** [RailsApps/rails3-mongoid-omniauth#22]
rake spec fails on new app.
to reproduce:

```
 rails new dummy -m https://raw.github.com/RailsApps/rails-composer/master/composer.rb -T -O
 cd dummy 
 rake spec
```

Finished in 0.09111 seconds
11 examples, 8 failures, 1 pending

**Comment by hiaw** [comment]
I find that this seems to help pass some of the test but not all
        @user = FactoryGirl.create(:user)
instead of 
        @user = create(:user)

**Comment by DanielKehoe** [comment]
I've now got all the specs passing except one.

```
rspec ./spec/controllers/users_controller_spec.rb:51 # UsersController PUT update with an error renders edit template
```

**Comment by MarkDBlackwell** [comment]
Fixed in pull request #23, I believe.
